<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveVideoDialog</class>
 <widget class="QWidget" name="SaveVideoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="labelImageView">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="ContentBoardItem" name="widgetDashBoardInsertNewVideoFile" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>88</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>88</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/*QFrame{
     background:rgb(249,247,248);
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 18px;
  max-height: 18px;
}

QLineEdit
{
    background:rgb(255,255,255);
    color:rgb(39,51,66);
    font-size: 16px;
    font-weight: bold;
    font-family: Siemens Sans;
    border: 1px solid rgb(39,51,66); 
    min-height: 32px;
    max-height: 32px;
}
*/
</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelMasseageNewVideoFile">
            <property name="font">
             <font>
              <family>Siemens Sans</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Insert New Video/File Name:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEditInputDataNewVideoFile">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="ContentBoardItem" name="widgetDashBoardVideoLocation" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>88</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>88</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/*QFrame{
     background:rgb(249,247,248);
     min-height: 80px;
     max-height: 80px;
}

QLabel{
  background:rgb(249,247,248);
  color: rgb(39,51,66);
  font-size: 16px;
  font-family: Siemens Sans;
  min-height: 18px;
  max-height: 18px;
}

QLineEdit
{
    background:rgb(255,255,255);
    color:rgb(39,51,66);
    font-size: 16px;
    font-weight: bold;
    font-family: Siemens Sans;
    border: 1px solid rgb(39,51,66); 
    min-height: 32px;
    max-height: 32px;
}
*/
</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelMasseageVideoLocation">
            <property name="font">
             <font>
              <family>Siemens Sans</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Video/File Location:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEditInputDataVideoLocation">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Siemens Sans</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ContentBoardItem</class>
   <extends>QWidget</extends>
   <header location="global">contentboarditem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
